core/server/models/plugins/collision.js

```diff
@@ -63,10 +63,9 @@ module.exports = function (Bookshelf) {
                                     message: 'Saving failed! Someone else is editing this post.',
                                     code: 'UPDATE_COLLISION',
                                     level: 'critical',
-                                    context: {
+                                    errorDetails: {
                                         clientUpdatedAt: self.clientData.updated_at,
-                                        serverUpdatedAt: self.serverData.updated_at,
-                                        changed: changed
```

Comment:
The `change` field was introduced temporarily here - https://github.com/TryGhost/Ghost/commit/30e790bf122da291d2b27f52cdde767a76386132#diff-eb942d2a7406b236df44c35a83722627 but then the comment was removed in https://github.com/TryGhost/Ghost/commit/3289dc76191ce90c3d1149610482134110171164#diff-eb942d2a7406b236df44c35a83722627 . Think we don't need to keep it here anymore.

Reply:
We've added the changed information for transparency and debugging purposes.